hub/api/tensor.py

```diff
@@ -95,24 +96,15 @@ def append(self, array: np.ndarray):
 
     @property
     def meta(self):
-        return read_tensor_meta(self.key, self.storage)
-
-    @meta.setter
-    def meta(self, new_meta: dict):
-        write_tensor_meta(self.key, self.storage, new_meta)
+        return TensorMeta.load(self.key, self.storage)
```

Comment:
this shouldn't be slow because we use cache chains with memory as the first part. without doing this the meta may be slightly desynchronized (in case there are multiple references floating around)

Reply:
Yup, agree. I was thought about this too but it'll be fine.